stdlib/public/TensorFlow/CMakeLists.txt

```diff
@@ -57,10 +57,31 @@ add_swift_target_library(swiftTensorFlow ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} IS_
   SWIFT_MODULE_DEPENDS_HAIKU Glibc
   SWIFT_MODULE_DEPENDS_WINDOWS MSVCRT
   SWIFT_MODULE_DEPENDS Python
-  SWIFT_COMPILE_FLAGS "${swift_stdlib_compile_flags}"
-  LINK_FLAGS "${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}"
+  SWIFT_COMPILE_FLAGS
+    ${swift_stdlib_compile_flags}
+    -Xcc -I${CMAKE_CURRENT_SOURCE_DIR}
+    -Xcc -I${TF_INCLUDE_DIR}
+  LINK_FLAGS
+    ${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}
+    -L${TensorFlow_LIBRARY_DIR}
   INSTALL_IN_COMPONENT stdlib
   EXTRA_RPATHS "${SWIFT_TENSORFLOW_TARGET_LIB_DIR}"
   # NOTE: This is a workaround for https://github.com/apple/swift/pull/24382,
   # which changed the default install_name_dir to `/usr/bin/swift`.
   DARWIN_INSTALL_NAME_DIR "@rpath")
+
```

Comment:
minor, feel free to ignore:
```suggestion
```
I personally try to uphold the "max one empty line" rule, across languages.

Reply:
I think that CMake really works better with multiple lines across the install/build phase to quickly identify where one ends and the other begins, especially with the completely custom implementation of CMake in CMake that Swift uses.